crates/solidity/testing/snapshots/cst_output/TrailingTrivia/only_single_comment/generated/0.4.11-success.yml

```diff
@@ -0,0 +1,17 @@
+# This file is generated automatically by infrastructure scripts. Please don't edit by hand.
+
+Source: >
+  1  │  // first                                                                        │ 0..9
+  2  │  // second                                                                       │ 10..20
+
+Errors: []
+
+Tree:
+  - TrailingTrivia: # 0..21 " // first\n // second\n"
```

Comment:
We only collect remaining trivia for the top-level parse (no-one really expects to top-level parse trailing trivia... right?). This should be special-cased (like trivia CST tests including whitespace tokens in the CST) to not do that and using the regular "SKIP over remaining unexpected input" logic.

Reply:
> no-one really expects to top-level parse trailing trivia... right?

They can now (via `language.parse(RuleKind.TrailingTrivia)`), but this is why I mentioned it will no longer be an issue once we remove it.
Does that match what you are thinking?